### ISSUE TITLE
cmdDeletion doesn't work anymore after the cmdEditing rewrite

### ISSUE THREAD

**Issue by Dev-Yukine** [dirigeants/klasa#259]
### Describe the issue
as the title already says the cmdDeletion doesn't work anymore after the cmdEditing rewrite.

### Code or steps to reproduce

```js
const { Client } = require('klasa');

new Client({
    prefix: '+',
    cmdDeleting: true,
    readyMessage: (client) => `Successfully initialized. Ready to serve ${client.guilds.size} guilds.`
}).login('your-bot-token');
```

### Expected an actual behaviour
Delete the message if invoke message is deleted

### Further details

- **discord.js version**: newest master (https://github.com/discordjs/discord.js/commit/a5e8f05d01e34b69b80736688704ee594b4d0900)
- **node.js version**: 9.10.1
- **Klasa version**: newest master (https://github.com/dirigeants/klasa/commit/7de01e1e2e69d9d0d8f51d171573653df7a2c362)
- [ ] I have modified core files.
- [x] I have tested the issue on latest master. Commit hash: 7de01e1e2e69d9d0d8f51d171573653df7a2c362

**Comment by Dev-Yukine** [comment]
fixed in 842e3d67361126b391ac221f309100914a425510

**Comment by kyranet** [comment]
Fixed in [`842e3d6`](https://github.com/dirigeants/klasa/commit/842e3d67361126b391ac221f309100914a425510)
